P1371R1.md

```diff
@@ -1522,19 +1526,19 @@ void Node<T>::balance() {
     //  (Red) x    c            a     b    c     d
     //   /    \
     //  a      b
-    [^Black, *[^Red, *[^Red, a, x, b], y, c], z, d]
+    [^Black, *![^Red, *[^Red, a, x, b], y, c], z, d]
       => Node{Red, std::make_shared<Node>(Black, a, x, b),
                    y,
                    std::make_shared<Node>(Black, c, z, d)},
-    [^Black, *[^Red, a, x, *[^Red, b, y, c]], z, d]  // left-right case
+    [^Black, *![^Red, a, x, *[^Red, b, y, c]], z, d]  // left-right case
       => Node{Red, std::make_shared<Node>(Black, a, x, b),
                    y,
                    std::make_shared<Node>(Black, c, z, d)},
-    [^Black, a, x, *[^Red, *[^Red, b, y, c], z, d]]  // right-left case
+    [^Black, a, x, *![^Red, *[^Red, b, y, c], z, d]]  // right-left case
       => Node{Red, std::make_shared<Node>(Black, a, x, b),
                    y,
                    std::make_shared<Node>(Black, c, z, d)},
-    [^Black, a, x, *[^Red, b, y, *[^Red, c, z, d]]]  // right-right case
+    [^Black, a, x, *![^Red, b, y, *[^Red, c, z, d]]]  // right-right case
```

Comment:
If we're adding the two symbol syntax, let's add it to the rest of the line too.

But looking at it now, looks busy as hell :(

Reply:
Also, these should be `*?` but I'll merge first and fix.